spec/supabase_kt_v0.yml

```diff
@@ -0,0 +1,3502 @@
+          ) {
+              install(GoTrue)
+              install(Postgrest)
+              //install other modules
+          }
+          ```
+      - id: configure-gotrue
+        name: Configure GoTrue module
+        code: |
+          ```kotlin
+          val client = createSupabaseClient(
+              supabaseUrl = "https://xyzcompany.supabase.co", 
+              supabaseKey = "public-anon-key"
+          ) {
+              install(GoTrue) {
+                  alwaysAutoRefresh = false // default: true
+                  autoLoadFromStorage = false // default: true
+                  //and more...
+              }
+          }
+          ```
+        description: |
+          **Common:**
+          
+          `alwaysAutoRefresh` - whether the GoTrue plugin should always auto refresh expired sessions automatically. Default: `true`
+          
+          `autoLoadFromStorage` - whether the GoTrue plugin should automatically load the session from the session manager. Default: `true`
+          
+          `flowType` - Which authentication flow to use. Currently available: FlowType.PKCE and FlowType.IMPLICIT. Default: `FlowType.IMPLICIT`
+          
+          `codeVerifierCache` - Interface for saving and loading codes for the PKCE authentication flow. Default: `SettingsCodeVerifierCache`
+          
+          `customUrl` - Custom url for the GoTrue Api. Can be safely ignored when using supabase. Default: `null`
+          
+          `jwtToken` - Plugin specific JWT Token. Can be ignored when using the GoTrue plugin. Default: `null`
+          
+          `retryDelay` - Duration after which the GoTrue plugin should retry a failed session refresh. Default: `10.seconds`
+          
+          `sessionManager` - Interface for saving and loading the user session. Default: `SettingsSessionManager`
+          
+          **Android & IOS:**
+          
+          `scheme` - The scheme for the redirect url, when using deep linking. Default: `supabase`
+          
+          `host` - The host for the redirect url, when using deep linking. Default: `login`
+          
+          **Android:** 
+          
+          `enableLifecycleCallbacks` - Whether to stop auto-refresh on focus loss, and resume it on focus again. Default: `true`
+          
+          **Desktop:**
+          
+          `httpPort`: The port the web server is running on, when logging in with OAuth. Default: `0` (random port).
+          
+          `timeout`: The timeout for the web server, when logging in with OAuth. Default: `1.minutes`.
+          
+          `htmlTitle`: The title of the redirect page, when logging in with OAuth. Default: `"Supabase Auth"`.
+          
+          `htmlText`: The text of the redirect page, when logging in with OAuth. Default: `"Logged in. You may continue in your app."`.
+          
+          `htmlIconUrl`: The icon of the redirect page, when logging in with OAuth. Default: `"https://supabase.com/brand-assets/supabase-logo-icon.png"`.
```

Comment:
This leads to a 404 - @gregnr @saltcod do we have a static URL for our logo?

Reply:
@jan-tennert Any ideas where this link came from originally? Also seeing it here:
https://github.com/supabase-community/supabase-kt/blob/c0b196dd4e12591d4a48a09ff9c564f02a5b9ac2/GoTrue/src/jvmMain/kotlin/io/github/jan/supabase/gotrue/GoTrueConfig.kt#L30